include/z64camera.h

```diff
@@ -1249,6 +1249,14 @@ typedef enum {
 
 #define ONEPOINT_CS_GET_ACTION(onePointCsFull) ((onePointCsFull)->actionFlags & 0x1F)
 
+#define ONEPOINT_CS_FIELD_TYPE_NONE 0xFF
+#define ONEPOINT_CS_FIELD_TYPE_MASK 0xF0
```

Comment:
```suggestion
#define ONEPOINT_CS_FIELD_NONE 0xFF
#define ONEPOINT_CS_FIELD_TYPE_MASK 0xF0
```
it seems it would make more sense to consider 0xFF as a full value indicating none rather than a type which you'd expect to only span some bits somehow?